docs/manual/src/python/setup.md

```diff
@@ -0,0 +1,31 @@
+# Intro
+
+The main idea is to build the bindings with commands.
+
+So you must create a setup.py file. This file will include the command to generate python .py bindings, link .dll and then build to pipy. Requires python version greater than 3.6.
+
+The full example is available at this [address](https://github.com/mozilla/uniffi-rs/tree/main/examples/distributing/uniffi-rust-to-python-library/src/setup.py).
+
+Once you reproducted the template on your project, fell free to run with:
+
+## Windows Powershell
+
+```powershell
+python .\setup.py bdist_wheel --verbose ;
+$wheelFile = Get-ChildItem -Path .\dist\ -Recurse -Include * ;
+pip3 install $wheelFile --force-reinstall ;
+```
+
+## Linux Bash
+
+```bash
+python .\setup.py bdist_wheel --verbose ;
```

Comment:
I think this is wrong due to the backslash and the use of "python" insetad of "python3". You can probably combine Linux and Mac too

Reply:
Absolutely. The single issue is that I set in the tutorial how to do something that works in cicd and on my computer. I absolutely want to do something tested on a computer.

Reply:
on cicd and my computer python3 has been installed with choco and this is the issue.

Reply:
by default python points to python3 on windows 10. Can I let as it is?